crates/egui/src/containers/collapsing_header.rs

```diff
@@ -174,7 +174,7 @@ impl CollapsingState {
     ) -> Option<InnerResponse<R>> {
         let id = self.id;
         self.show_body_unindented(ui, |ui| {
-            ui.indent(id, |ui| {
+            ui.indent(|ui| {
```

Comment:
I haven't looked to closely at this yet, but we need to make sure the id of `CollapsingHeader` is not ignored. This is so that two collapsing headers in the same ui won't have an id clash.

Perhaps we need a `indent_with_id_source` just for this use case, or add an extra `ui.push_id` call here